### PR TITLE
closes #398 - "Previous/Next Surah" when reading first/last surah

### LINKED ISSUE #398: "Previous/Next Surah" when reading first/last surah.
If you tried to click on Previous Surah while reading Surah Al-Fatihah or to click on Next Surah while reading Surah An-Nas, you will get message:

> Surah is out of range. Please go to thehome page and select a Surah/Ayah

Can we hide/disable that button?

The location of the button:
previous surah button in Al-Fatihah

### PR DESCRIPTION
**Pull request by sabeurthabti**
Fixed + unit tests

closes #398

### CODE REVIEW

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32831

File: src/containers/Surah/Title/spec.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { shallow, render } from 'enzyme';
```
Comment (mmahalwy) [inline src/containers/Surah/Title/spec.js]: i dont think you are using `render`

File: src/containers/Surah/Title/spec.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    const previous = wrapper.find('.previous-chapter').length;
+    const next = wrapper.find('.next-chapter').length;
+
+    expect(previous).to.equal(1);
+    expect(next).to.equal(1);
+  });
+});
+
+function renderComponent(data) {
+  return shallow(<Title surah={data}/>)
```
Comment (mmahalwy) [inline src/containers/Surah/Title/spec.js]: semicolon ;)

**Comment by mmahalwy** [comment]
woot! This is awesome mashallah :)

**Comment by sabeurthabti** [comment]
fixes have been pushed.

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32832

File: package.json [hunk 3]
```diff
@@ -3,11 +3,11 @@
   "version": "1.0.0",
   "private": false,
   "scripts": {
-    "test": "./node_modules/karma/bin/karma start",
```
Comment (sabeurthabti) [inline package.json]: just an FYI, npm makes anything within node_modules/.bin usable as a global within npm scripts.

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32839

**Comment by sabeurthabti** [comment]
can we get this merged in?

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32840

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32842

**Comment by ahmedre** [comment]
merged based on your previous comment @mmahalwy

**Comment by srifqi** [comment]
Maybe late, but seems like changing the JavaScript code is not enough. I mean you just changes from `<=` into `<`. But it is already exist in HTML code and don't get something like `display: none;` in its style. I don't know much about this project's code, so I can't help much.

**Comment by sabeurthabti** [comment]
@srifqi I don't think there was a production release.

**Pull request merged**